ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -108,15 +108,11 @@ section {
 
             &__title {
                 display: inline;
-                font-family: $primary-font-medium;
-                letter-spacing: -0.07px;
-                line-height: 24px;
-                margin-right: 8px;
+                @extend .participate-step-label;
             }
 
             &__time {
```

Comment:
looks like we can remove class step__time and use .participate-step-time class on the element

Reply:
Fixed at 316abf2f0e57626829a3c6149f4cf9089db50904